lib/rubocop/cop/style/space_around_keyword.rb

```diff
@@ -0,0 +1,192 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks the spacing around the keywords.
+      #
+      # @example
```

Comment:
I'd add a few more examples here - e.g. `while` and some modifier, so it's clear this handles all sorts of spacing around keywords.